processccd.rst

```diff
@@ -1,131 +1,100 @@
 
-2/4
 
-ProcessCCD
-================
+##############
+ProcessCcdTask
+##############
 
+ProcessCcd (which is a ``Cmd Line Task``, and thus can be run directly
```

Comment:
Ok - am keeping it in backticks, assuming it will link to proper destination later when needed, yes?  Also, as it's 'special' i had it capitalized, but is the lower case preferred?